qdrant-landing/content/documentation/fastembed/fastembed-splade.md

```diff
@@ -0,0 +1,196 @@
+weight: 5
+---
+
+# How to Generate Sparse Vectors with SPLADE
+
+SPLADE is a novel method for learning sparse text representation vectors, outperforming BM25 in tasks like information retrieval and document classification. Its main advantage is generating efficient and interpretable sparse vectors, making it effective for large-scale text data.
+
+## Setup
+
+First, install FastEmbed.
+
+```python
+pip install -q fastembed
+``` 
+
+Next, import the required modules for sparse embeddings and Python’s typing module.
+
+```python
+from fastembed import SparseTextEmbedding, SparseEmbedding
+from typing import List
+```
+
+You may always check the list of all supported sparse embedding models.
+
+```python
+SparseTextEmbedding.list_supported_models()
+```
+This will return a list of models, each with its details such as model name, vocabulary size, description, and sources.
+
+```python
+[{'model': 'prithivida/Splade_PP_en_v1',
+  'vocab_size': 30522,
+  'description': 'Independent Implementation of SPLADE++ Model for English',
+  'size_in_GB': 0.532,
+  'sources': {'hf': 'Qdrant/SPLADE_PP_en_v1'}}]
+```
+
+Now, load the model.
+
+```python
+model_name = "prithvida/Splade_PP_en_v1"
+# This triggers the model download
+model = SparseTextEmbedding(model_name=model_name)
+```
+## Embed data
+
+You need to define a list of documents to be embedded.
+```python
+documents: List[str] = [
+    "Chandrayaan-3 is India's third lunar mission",
+    "It aimed to land a rover on the Moon's surface - joining the US, China and Russia",
+    "The mission is a follow-up to Chandrayaan-2, which had partial success",
+    "Chandrayaan-3 will be launched by the Indian Space Research Organisation (ISRO)",
+    "The estimated cost of the mission is around $35 million",
+    "It will carry instruments to study the lunar surface and atmosphere",
+    "Chandrayaan-3 landed on the Moon's surface on 23rd August 2023",
+    "It consists of a lander named Vikram and a rover named Pragyan similar to Chandrayaan-2. Its propulsion module would act like an orbiter.",
+    "The propulsion module carries the lander and rover configuration until the spacecraft is in a 100-kilometre (62 mi) lunar orbit",
+    "The mission used GSLV Mk III rocket for its launch",
+    "Chandrayaan-3 was launched from the Satish Dhawan Space Centre in Sriharikota",
+    "Chandrayaan-3 was launched earlier in the year 2023",
+]
+```
+Then, generate sparse embeddings for each document. 
+Here,`batch_size` is optional and helps to process documents in batches.
+
+```python
+sparse_embeddings_list: List[SparseEmbedding] = list(
+    model.embed(documents, batch_size=6)
+) 
+```
+## Retrieve embeddings
+
+`Sparse_embeddings_list` contains sparse embeddings for the documents provided earlier. Each element in this list is a `SparseEmbedding` object that contains the sparse vector representation of a document. Set the variable `index` to 0. This will retrieve data from the first document in the list.
```

Comment:
```suggestion
`sparse_embeddings_list` contains sparse embeddings for the documents provided earlier. Each element in this list is a `SparseEmbedding` object that contains the sparse vector representation of a document. Set the variable `index` to 0. This will retrieve data from the first document in the list.
```